### PR TITLE
Sbank 195 confirm registration

### PR DESCRIPTION
**Pull request by Parker-Williams-Smoothstack**

### CODE REVIEW

File: src/components/CardRegistration.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+    schema = {
+        name: Joi.string().required().label("Your name"),
+        email: Joi.string().required().email({ minDomainAtons: 2 }).label("Email"),
+        home_phone: Joi.string().min(7).label("Home phone"),
+        cell_phone: Joi.string().min(7).label("Cell phone"),
+        work_phone: Joi.string().min(7).label("Work phone"),
+        address: Joi.string().required().label("Mailing address"),
+        po_box: Joi.string().label("P.O box"),
+        zipcode: Joi.number().required().label("Zipcode"),
+        ssn: Joi.string().min(10).max(10).required().label("Social Security Number"),
```
Comment (ghost) [inline src/components/CardRegistration.js]: If I'm not mistaken, isn't SSN 9 digits?

Comment (Parker-Williams-Smoothstack) [reply src/components/CardRegistration.js]: SSN is nine digits, 3 area digits 2 Group Number digits and 4 Serial Number digits according to https://www.ssa.gov/history/ssn/geocard.html#:~:text=The%20nine-digit%20SSN%20is%20composed%20of%20three%20parts%3A,set%20of%20four%20digits%20is%20the%20Serial%20Number.

Comment (ghost) [reply src/components/CardRegistration.js]: I suggest changing the phone numbers to 10 that way area code is always needed, and then ssn can change to min(9).max(9)

Comment (Parker-Williams-Smoothstack) [reply src/components/CardRegistration.js]: Made the changes, so it should now use the correct number of digits for SSN and the phone numbers now ask for the area code (10 digits up from 7 digits).

**Review by szahedi-smoothstack — APPROVED** [review]
Looks good

**Pull request merged**